src/backend/router/v2/meeting.js

```diff
@@ -22,5 +23,12 @@ module.exports = () => {
     res.json(makedUser);
```

Comment:
makedUser...? I thought you'd changed this name in the prior commit.
It all looks good except for this one. Feel free to merge when this one is changed 😃